galleon-pack/src/main/resources/layers/standalone/jaxrs-server/layer-spec.xml

```diff
@@ -2,7 +2,7 @@
 <layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="jaxrs-server">
     <dependencies>
         <layer name="datasources-web-server"/>
-        <layer name="jaxrs" />
+        <layer name="jaxrs" optional="true"/>
```

Comment:
A comment here as to why this is optional would be good. Some maintainer some day will wonder and may naively remove the optional.

Reply:
Ok, added.